generator/osm2type.cpp

```diff
@@ -1252,15 +1306,12 @@ void GetNameAndType(OsmElement * p, FeatureBuilderParams & params,
   {
     std::string const typesString = params.PrintTypes();
 
-    // Unknown type is possible in unit tests.
-    std::string const id = (p->m_type != OsmElement::EntityType::Unknown) ?
-          DebugPrint(GetGeoObjectId(*p)) : std::string("Unknown");
-
     if (params.RemoveInconsistentTypes())
-      LOG(LWARNING, ("Inconsistent types for:", id, "Types:", typesString));
+      LOG(LWARNING, ("Inconsistent types for:", DebugPrintID(*p), "Types:", typesString));
 
+    size_t const typesCount = params.m_types.size();
     if (params.FinishAddingTypesEx() == FeatureParams::TYPES_EXCEED_MAX)
-      LOG(LWARNING, ("Exceeded types count for:", id, "Types:", typesString));
+      LOG(LWARNING, ("Exceeded types count for:", DebugPrintID(*p), "Types:", typesCount, typesString));
```

Comment:
please put `typesCount =` under the if

Reply:
But the purpose is to know the initial exceeded types count. Under the "if" it will be already truncated = 7.

Reply:
Right, I've missed it